docs/versioning.rst

```diff
@@ -0,0 +1,96 @@
+**********
+Versioning
+**********
+
+
+Current Status
+==============
+
+When a new major version is released it becomes the current release line, where all new features, bug fixes and security patches land.
+
+
+.. list-table:: Release Lines
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Line
+     - Supported Runtimes
+     - Minimum Trace Agent
```

Comment:
Should we recommend using the latest version of the trace agent? Certain features (ex. telemetry) are only supported in recent trace agent release. Here we could add a `Recommended trace agent` column

Reply:
that is what the minimum version is meant to help with. I could see us adding a footnote/aside that says "we always recommend using the latest version of the trace agent". but I would want us to say "min version is 7.22 but recommended is 7.35" that sends mixed messages.